osgbuild/main.py

```diff
@@ -71,19 +71,14 @@ def main(argv):
     if task == 'koji' and buildopts['vcs']:
         # verify working dirs
         for pkg in package_dirs:
-            git_ok = False
-            try:
-                git_ok = git.is_git(pkg) and git.verify_working_dir(pkg)
-            except GitError as err:
-                log.info(str(err))
-            svn_ok = False
-            try:
-                svn_ok = svn.is_svn(pkg) and svn.verify_working_dir(pkg)
-            except SVNError as err:
-                log.info(str(err))
-
-            vcs = (git_ok and git) or (svn_ok and svn)
-            if not vcs:
+            vcs_ok = False
+            vcs = (svn.is_svn(pkg) and svn) or (git.is_git(pkg) and git)
```

Comment:
Em, i realize you didn't add these, but what are `svn` and `git` in boolean context?

Reply:
They're modules so they're not False.

Since we've dropped Python 2.4 I could change it to something like
```
vcs = svn if svn.is_svn(pkg) else git if git.is_git(pkg) else None
```

Reply:
ah, ok - yeah the point i missed is that `vcs` is used as a module rather than a boolean.